### DISCUSSION TITLE
Issues building war

### DISCUSSION THREAD

**Discussion by C-Kan-42** [Caltech-IPAC/firefly#1164] [Q&A]
Hello! I'm a developer at the HEASARC and just started looking into Firefly. I don't have much experience with Gradle, but I am running into an error when building the war file (I'm using Gradle 7.3.1). 

To reproduce:

While in local Firefly directory, run

`gradle :firefly:warAll`

This is the error stack trace:

```
 > Task :firefly:prepareWebapp FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':firefly:prepareWebapp'.
> Could not copy file '/Users/cdkan/Desktop/firefly/config/logging.properties' to '/Users/cdkan/Desktop/firefly/build/firefly/war/WEB-INF/config/logging.properties'.
   > java.util.ArrayList cannot be cast to java.lang.String
```

I'm able to manually copy the file successfully. Any help would be appreciated, thank you!

**Comment by robyww** [comment]
@loitly is out until January. He can probably answer this question.

We can help you build Firefly but why not just use the docker image?

**Reply by C-Kan-42** [reply]
> We can help you build Firefly but why not just use the docker image?

I will try that, thank you!

**Comment by robyww** [comment]
https://github.com/Caltech-IPAC/firefly/blob/dev/docs/firefly-docker.md
